.build/BuildScriptVersions.ps1

```diff
@@ -0,0 +1,34 @@
+﻿# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+[CmdletBinding()]
+param (
+
+)
+
+#Requires -Version 7
+
+$repoRoot = Get-Item "$PSScriptRoot\.."
+$distFolder = "$repoRoot\dist"
+$scriptVersionsCsv = "$distFolder\ScriptVersions.csv"
+
+if (Test-Path -Path $scriptVersionsCsv) {
+    $versionsFileCSV = ConvertFrom-Csv -InputObject (Get-Content -Path $scriptVersionsCsv)
+
+    # Generate final ScriptVersions.txt file (with SHA256 hash) for release description
+    $versionFile = "$distFolder\ScriptVersions.txt"
+    New-Item -Path $versionFile -ItemType File -Force | Out-Null
+    "Script | Version | SHA256 Hash" | Out-File $versionFile -Append
+    "-------|---------|------------" | Out-File $versionFile -Append
+    foreach ($script in $versionsFileCSV) {
+        $sha256Hash = $((Get-FileHash -Path "$($distFolder)\$($script.File)").Hash)
+        $script | Add-Member -MemberType NoteProperty -Name SHA256Hash -Value $sha256Hash
+        "$($script.File) | $($script.Version) | $sha256Hash" | Out-File $versionFile -Append
+        Write-Host ("File: '{0}' Version: '{1}' Hash: '{2}' added" -f $script.File, $script.Version, $sha256Hash)
+    }
+
+    $versionsFileCSV | Export-Csv -Path $scriptVersionsCsv
```

Comment:
This should include `-NoTypeInformation` like the original csv file.

Reply:
It already wasn't adding type information, maybe because these objects were PSObject because they were imported from CSV:

![image](https://user-images.githubusercontent.com/4518572/155740423-9d18d470-2388-4e63-927c-73cfab0b0876.png)

But I added it anyway.